core/src/main/java/org/mapfish/print/config/CustomFontLoader.java

```diff
@@ -0,0 +1,77 @@
+package org.mapfish.print.config;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.awt.FontFormatException;
+import java.awt.GraphicsEnvironment;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Set;
+
+import javax.annotation.PostConstruct;
+
+
+/**
+ * Used to load custom fonts.
+ *
+ * To add a custom font, the file `mapfish-spring-custom-fonts.xml` must be overridden.
+ */
+public final class CustomFontLoader {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CustomFontLoader.class);
+
+    private final Set<String> paths;
+
+    /**
+     * Constructor.
+     * @param paths A list of paths to ttf font files.
+     */
+    public CustomFontLoader(final Set<String> paths) {
+        this.paths = paths;
+    }
+
+    /**
+     * Load the custom fonts when the application is started.
+     */
+    @PostConstruct
+    public void init() {
+        for (String path : this.paths) {
```

Comment:
Why don't you do the loading in the constructor? I thought you wanted to do all that ASAP?